src/backend/sideload/games.ts

```diff
@@ -122,7 +122,7 @@ export async function launchApp(appName: string): Promise<boolean> {
       mangoHudCommand,
       gameModeBin,
       steamRuntime?.length
-        ? [...steamRuntime, `--filesystem=${gameInfo.install.install_path}`]
+        ? [...steamRuntime, `--filesystem=${gameInfo.folder_name}`]
```

Comment:
This isn't right. `gameInfo.folder_name` contains only the game folder name (either `metadata.customAttributes.FolderName.value` or the game's AppName if the former is not set). E.g. let's say I install Fortnite, `gameInfo.folder_name` only contains `Fortnite`

Reply:
Yes, this is only for Sideload Games, I a not changing for the others.
On this one, we don't have any metadata so I added the folder name on the main key object.

Reply:
Hm, I see. I'd still say that we should keep this in line with other metadata (so setting `install.install_path` to the path & maybe leaving `folder_name` empty), but I guess this works for now